### PR TITLE
"add title & description"

### PR DESCRIPTION
**Pull request by Subrat29**
Add title and description in about.mdx.
can i add know more(link) or problem id in it??

### CODE REVIEW

File: app/aboutproject/about.mdx [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
- 
-- One
-- Two
-- Three
- 
-## Checkout my React component:
-
-* This is a list in markdown:
+# Problem Statement Title
+### Identification of Different Medicinal Plants/Raw materials through Image Processing Using Machine Learning Algorithms
```
Comment (anurag629) [inline app/aboutproject/about.mdx]: Leave a line between lines first and second. Make the statement in `**Identification of Different...**` 


Look through [MDX Guide](https://kabartolo.github.io/chicago-docs-demo/docs/mdx-guide/writing/)

File: app/aboutproject/about.mdx [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
+***
+# Description	
+* India, with a rich heritage of floral diversity, is well-known for its medicinal plant wealth, but their identification is one of the major burning issues in Ayurvedic Pharmaceutics. Several crude drugs are being sold under the same name in the market leading to confusion and their misidentification. Even the collectors and traders are not completely aware of the exact morphological appearance or differentiating attributes of the many drugs owing to seasonal and geographical availability, and similar characteristics. Moreover, the extensive consumption to meet demand-supply ratio exerts a heavy strain on the existing resources. It further leads to the practice of adulteration, substitution, and disbelief in the curative capability of the system eventually. Thus, software capable of identifying different medicinal plants/ raw materials through Image Processing Using Different Machine Learning Algorithms will be of immense use. It will be helpful at every level viz. wholesaler, distributor, etc. of the supply chain of raw material being utilized in the system.
```
Comment (anurag629) [inline app/aboutproject/about.mdx]: Same leave a line between line 4 and 5

**Review by anurag629 — CHANGES_REQUESTED** [review]
Also, change this description according to statement

image

File: app/aboutproject/about.mdx (hunk 1 above)
Comment (Subrat29) [reply app/aboutproject/about.mdx]: ok done

File: app/aboutproject/about.mdx (hunk 2 above)
Comment (Subrat29) [reply app/aboutproject/about.mdx]: ok done

**Review by anurag629 — APPROVED** [review]

**Pull request merged**